### PR TITLE
[PRM-356] 

### PR DESCRIPTION
**Pull request by Chris-Williams-QA**

### CODE REVIEW

File: test/forms/WhyReturnSubmittedLateFormSpec.scala [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import base.SpecBase
+import play.api.data.FormError
+
+class WhyReturnSubmittedLateFormSpec extends SpecBase {
+  val form = WhyReturnSubmittedLateForm.whyReturnSubmittedLateForm()
+
+  "the value entered does not exist in the possible, valid values" in {
+    val result = form.bind(Map("why-return-submitted-late-text" -> "")).apply("why-return-submitted-late-text")
+    result.errors.headOption shouldBe Some(FormError("why-return-submitted-late-text", "otherReason.whyReturnSubmittedLate.error.required"))
+  }
```
Comment (AndrewRButcher) [inline test/forms/WhyReturnSubmittedLateFormSpec.scala:28]: Can we change to:
```suggestion
  "When no reason is given" should {
  "give required error and not bind" in {
    val result = form.bind(Map("why-return-submitted-late-text" -> "")).apply("why-return-submitted-late-text")
    result.errors.headOption shouldBe Some(FormError("why-return-submitted-late-text", "otherReason.whyReturnSubmittedLate.error.required"))
    }
  }
```

File: test/forms/WhyReturnSubmittedLateFormSpec.scala [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ */
+
+package forms
+
+import base.SpecBase
+import play.api.data.FormError
+
+class WhyReturnSubmittedLateFormSpec extends SpecBase {
+  val form = WhyReturnSubmittedLateForm.whyReturnSubmittedLateForm()
+
```
Comment (AndrewRButcher) [inline test/forms/WhyReturnSubmittedLateFormSpec.scala:24]: can we add a test for successful binding?
```suggestion
  "WhyReturnSubmittedLateForm" should {
    "bind when a valid reason is given" in {
    ....
    }
```

**Review by AndrewRButcher — CHANGES_REQUESTED** [review]
Needs rebasing + couple of comments on form tests

File: test/forms/WhyReturnSubmittedLateFormSpec.scala [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  "When no reason is given" should {
+    "give required error and not bind" in {
+      val result = form.bind(Map("why-return-submitted-late-text" -> "")).apply("why-return-submitted-late-text")
+      result.errors.headOption shouldBe Some(FormError("why-return-submitted-late-text", "otherReason.whyReturnSubmittedLate.error.required"))
+    }
+  }
+
+  "A valid reason is given" should {
+    "give required error and not bind" in {
```
Comment (AndrewRButcher) [inline test/forms/WhyReturnSubmittedLateFormSpec.scala:33]: Should be
```suggestion
    "bind successfully and not give errors" in {
```

**Review by AndrewRButcher — CHANGES_REQUESTED** [review]

File: test/forms/WhyReturnSubmittedLateFormSpec.scala (hunk 3 above)
Comment (Chris-Williams-QA) [reply test/forms/WhyReturnSubmittedLateFormSpec.scala]: Good catch! :)

**Pull request merged**